colorclusters/closest_color.py

```diff
@@ -27,18 +27,26 @@ def get_closest_color_index(pixel, colors, distance=euclidean):
     return index
 
 
-def map_pixels_to_closest_color_index(pixels, colors, distance=euclidean):
+def map_pixels_to_closest_color_index(pixels, colors, output_queue=None, distance=euclidean):
```

Comment:
Pre-existing code that explicitly states the distance function will now misinterpret it as being an output queue.
Can be fixed by switching it to:
... , distance=euclidean, output_queue=None):